integrationtests/k8s_test.go

```diff
@@ -74,6 +79,8 @@ func TestK8s(t *testing.T) {
 		ok = TestConnection(workerIP, "22")
 		assert.True(t, ok)
 
+		time.Sleep(10 * time.Second)
```

Comment:
why you sleep here?

Reply:
For some reason the command fails if I don't wait enough, fa I added sleep, not sure if the problem is connection in my side or what

Reply:
add it in the assert nodes function itself